exercises/hexadecimal/hexadecimal.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Hexadecimal do
+  @doc """
+    Accept a string representing a hexadecimal value and returns the
+    corresponding decimal value.
+    It returns the integer 0 if the hexadecimal is invalid.
+    Otherwise returns an integer representing the decimal value.
+
+    ## Examples
+
+      iex> Hexadecimal.to_decimal("invalid")
+      0
+
+      iex> Hexadecimal.to_decimal("af")
+      175
+
+  """
+
+  @spec to_decimal(binary) :: integer
+  def to_decimal(hex) do
+
```

Comment:
Hello world had given me something like "Your code goes here" so it wasn't known to me if it was a established pattern to indicate that.